mesh/v1alpha1/config.proto

```diff
@@ -133,6 +133,12 @@ message ProxyConfig {
 
   // The mode used to redirect inbound traffic to Envoy.
   InboundInterceptionMode interception_mode = 18;
+
+  // Address of the LightStep service (e.g. _lightstep:80_).
+  string lightstep_address = 19;
+
+  // Path to the lightstep access token file.
+  string lightstep_access_token_file = 20;
```

Comment:
I would like us to first figure out exactly how we are going to mount the access_token_file into the envoy container. In my prototype I failed to do so and instead passed it in plaintext. I think we can do better. We probably will need to do something like this in order to provide the satellite cert https://github.com/lightstep/istio/pull/2#discussion_r216783954